### PR TITLE
LPS-184244 Distinguish between background and font color

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
https://issues.liferay.com/browse/LPS-184244

image

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by jaimelr10** [comment]
@victorg1991 Could you double check this?

File: modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/css/item_selector_preview.scss [hunk 1]
```diff
@@ -76,7 +76,8 @@ html#{$cadmin-selector}:not(._) {
 				}
 
 				.nav-link.active {
-					color: $cadmin-white;
+					background-color: $cadmin-white;
+					color: #272833;
```
Comment (victorg1991) [inline modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/css/item_selector_preview.scss:80]: We should not be using specific colors here but sass variables instead in this case we should be using $cadmin-dark. More info about the colors and variables can be found here: https://clayui.com/docs/css/color.html#clay-sidebar

Comment (AliciaGarciaGarcia) [reply modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/css/item_selector_preview.scss:80]: Thanks!, I will change it.
I saw this literal color on some sites but I didn't know what was the change.

Thanks!!!

<3

Comment (AliciaGarciaGarcia) [reply modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/css/item_selector_preview.scss:80]: Hi @victorg1991 
Are you sure that the clay variables are in the code?

I changed it and the SF (local) returned :

```
Error: Undefined variable.
   ╷
80 │                     color: $gray-900;
   │                            ^^^^^^^^^
   ╵
  src/main/resources/META-INF/resources/css/item_selector_preview.scss 80:13  root stylesheet
    at Object._newRenderError (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:13537:19)
    at Object._wrapException (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:13374:16)
    at StaticClosure._renderSync (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:13349:18)
    at Object.Primitives_applyFunction (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:1089:30)
    at Object.Function_apply (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:6007:16)
    at _callDartFunctionFast (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:7676:16)
    at Object.renderSync (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:7654:18)
    at buildSass (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/src/sass/build.js:56:22)
    at main (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/src/sass/build.js:190:34)
    at module.exports (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/src/scripts/build.js:328:4)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at Object.build (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/src/index.js:21:4)
    at module.exports (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/src/index.js:105:4)
    at main (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/bin/liferay-npm-scripts.js:12:2)
 {
  formatted: 'Error: Undefined variable.\n' +
    '   ╷\n' +
    '80 │                     color: $gray-900;\n' +
    '   │                            ^^^^^^^^^\n' +
    '   ╵\n' +
    '  src/main/resources/META-INF/resources/css/item_selector_preview.scss 80:13  root stylesheet',
```

Thanks for the time

Comment (AliciaGarciaGarcia) [reply modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/css/item_selector_preview.scss:80]: > Hi @victorg1991 Are you sure that the clay variables are in the code?
> 
> I changed it and the SF (local) returned :
> 
> ```
> Error: Undefined variable.
>    ╷
> 80 │                     color: $gray-900;
>    │                            ^^^^^^^^^
>    ╵
>   src/main/resources/META-INF/resources/css/item_selector_preview.scss 80:13  root stylesheet
>     at Object._newRenderError (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:13537:19)
>     at Object._wrapException (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:13374:16)
>     at StaticClosure._renderSync (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:13349:18)
>     at Object.Primitives_applyFunction (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:1089:30)
>     at Object.Function_apply (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:6007:16)
>     at _callDartFunctionFast (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:7676:16)
>     at Object.renderSync (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/sass/sass.dart.js:7654:18)
>     at buildSass (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/src/sass/build.js:56:22)
>     at main (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/src/sass/build.js:190:34)
>     at module.exports (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/src/scripts/build.js:328:4)
>     at processTicksAndRejections (node:internal/process/task_queues:96:5)
>     at Object.build (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/src/index.js:21:4)
>     at module.exports (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/src/index.js:105:4)
>     at main (/Users/aliciagarcia/liferay/ce/portal/modules/node_modules/@liferay/npm-scripts/bin/liferay-npm-scripts.js:12:2)
>  {
>   formatted: 'Error: Undefined variable.\n' +
>     '   ╷\n' +
>     '80 │                     color: $gray-900;\n' +
>     '   │                            ^^^^^^^^^\n' +
>     '   ╵\n' +
>     '  src/main/resources/META-INF/resources/css/item_selector_preview.scss 80:13  root stylesheet',
> ```
> 
> Or maybe do I have to add some "thing" on some "place"?
> 
> Thanks for the time

News:

I realized that there is 2 set of variables
```
@import 'atlas-variables';
@import 'cadmin-variables';
```

On this file we use 'cadmin-variables' so I will still use the styles on this.

I do not know if this are the old o something similar, but I will still use it till Clay decide the proper one.

thanks for all!

**Comment by AliciaGarciaGarcia** [comment]
made some changes and rebased

Thanks!

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/css/item_selector_preview.scss (hunk 1 above)
Comment (victorg1991) [reply modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/css/item_selector_preview.scss]: The two variables are valid, one are using in the context of cadmin (admin pages in the portal displayed in public pages) and the other when we are using clay only :)

**Comment by jaimelr10** [comment]
ci:forward

**Pull request closed without merging**